tx/textual/internal/testdata/timestamp.json

```diff
@@ -0,0 +1,14 @@
+[
+    [
+        {"seconds": "1136214245"},
+        "2006-01-02T15:04:05Z"
+    ],
+    [
+        {"seconds": "1136214245", "nanos": 123456789},
+        "2006-01-02T15:04:05.123456789Z"
+    ],
+    [
+        {"seconds": "1136214245", "nanos": 123000000},
+        "2006-01-02T15:04:05.123Z"
+    ]
+]
```

Comment:
Can you use the tests from https://github.com/cosmos/cosmjs/pull/1147/files#diff-1aee49f4f44f595fc5e4c8a2f0e15fce0b30fab5ea6e9e8f1b2894a917b19fa5?

Reply:
Done. Dropped a duplicate and also dropped the out-of-range tests, as the straightforward Go implementation doesn't return errors for those cases. It looks like the underlying Go library is able to tolerate slightly out-of-spec inputs, and that shouldn't be an error, IMHO.

Reply:
For these test vectors: I find having JSON objects with field names better for readability (especially for newcomers) than arrays 👍